cardano-testnet/src/Testnet/Babbage.hs

```diff
@@ -0,0 +1,416 @@
+    , "softforkRule" .= object
+      [ "initThd" .= ("900000000000000" :: String)
+      , "minThd" .= ("600000000000000" :: String)
+      , "thdDecrement" .= ("50000000000000" :: String)
+      ]
+    , "txFeePolicy" .= object
+      [ "multiplier" .= ("43946000000" :: String)
+      , "summand" .= ("155381000000000" :: String)
+      ]
+    ]
+
+  void $ H.note OS.os
+  currentTime <- H.noteShowIO DTC.getCurrentTime
+  startTime <- H.noteShow $ DTC.addUTCTime startTimeOffsetSeconds currentTime
+
+  void . H.execCli $
+    [ "byron", "genesis", "genesis"
+    , "--protocol-magic", show @Int testnetMagic
+    , "--start-time", showUTCTimeSeconds startTime
+    , "--k", show @Int (securityParam testnetOptions)
+    , "--n-poor-addresses", "0"
+    , "--n-delegate-addresses", show @Int (numSpoNodes testnetOptions)
+    , "--total-balance", show @Int (totalBalance testnetOptions)
+    , "--delegate-share", "1"
+    , "--avvm-entry-count", "0"
+    , "--avvm-entry-balance", "0"
+    , "--protocol-parameters-file", tempAbsPath </> "byron.genesis.spec.json"
+    , "--genesis-output-dir", tempAbsPath </> "byron-gen-command"
+    ]
+
+  -- Because in Babbage the overlay schedule and decentralization parameter
+  -- are deprecated, we must use the "create-staked" cli command to create
+  -- SPOs in the ShelleyGenesis
+
+  alonzoBabbageTestGenesisJsonSourceFile <- H.noteShow $ base </> "scripts/babbage/alonzo-babbage-test-genesis.json"
+  alonzoBabbageTestGenesisJsonTargetFile <- H.noteShow $ tempAbsPath </> "genesis.alonzo.spec.json"
+
+  H.copyFile alonzoBabbageTestGenesisJsonSourceFile alonzoBabbageTestGenesisJsonTargetFile
+
+  configurationFile <- H.noteShow $ tempAbsPath </> "configuration.yaml"
+
+  H.readFile configurationTemplate >>= H.writeFile configurationFile
+
+  H.rewriteYamlFile (tempAbsPath </> "configuration.yaml") . J.rewriteObject
+    $ HM.delete "GenesisFile"
+    . HM.insert "Protocol" (toJSON @String "Cardano")
+    . HM.insert "PBftSignatureThreshold" (toJSON @Double 0.6)
+    . HM.insert "minSeverity" (toJSON @String "Debug")
+    . HM.insert "ByronGenesisFile" (toJSON @String "genesis/byron/genesis.json")
+    . HM.insert "ShelleyGenesisFile" (toJSON @String "genesis/shelley/genesis.json")
+    . HM.insert "AlonzoGenesisFile" (toJSON @String "genesis/shelley/genesis.alonzo.json")
+    . HM.insert "RequiresNetworkMagic" (toJSON @String "RequiresMagic")
+    . HM.insert "LastKnownBlockVersion-Major" (toJSON @Int 6)
+    . HM.insert "LastKnownBlockVersion-Minor" (toJSON @Int 0)
+    . HM.insert "TestShelleyHardForkAtEpoch" (toJSON @Int 0)
+    . HM.insert "TestAllegraHardForkAtEpoch" (toJSON @Int 0)
+    . HM.insert "TestMaryHardForkAtEpoch" (toJSON @Int 0)
+    . HM.insert "TestAlonzoHardForkAtEpoch" (toJSON @Int 0)
+    . HM.insert "TestBabbageHardForkAtEpoch" (toJSON @Int 0)
+    . HM.insert "TestEnableDevelopmentHardForkEras" (toJSON True)
+    . HM.insert "TestEnableDevelopmentNetworkProtocols" (toJSON True)
```

Comment:
Supposedly `TestEnableDevelopmentNetworkProtocols` is deprecated. Can you check with the network team?

Reply:
Deleted.